device-types/Opengear/ACM7008-2-L.yaml

```diff
@@ -0,0 +1,33 @@
+---
+manufacturer: Opengear
+model: ACM7008-2-L
+slug: acm7008-2-l
+part_number: ACM7008-2-L
+u_height: 1
+is_full_depth: false
+comments: ACM7008 Console Server with 8 Console Ports on the rear of the device.
+console-server-ports:
+  - name: Port 1
+    type: rj-45
+  - name: Port 2
+    type: rj-45
+  - name: Port 3
+    type: rj-45
+  - name: Port 4
+    type: rj-45
+  - name: Port 5
+    type: rj-45
+  - name: Port 6
+    type: rj-45
+  - name: Port 7
+    type: rj-45
+  - name: Port 8
+    type: rj-45
+power-ports:
```

Comment:
Is [this image](https://webobjects2.cdw.com/is/image/CDW/5651010d?$product-detail$) the right model? 

If so, please make sure the power type is correct. c14 is definitely not correct.